.travis.yml

```diff
@@ -63,7 +63,6 @@ matrix:
       rust: stable
       script:
         - cargo build
```

Comment:
I'm not 100% suree, but I do not think this does the right thing. This will build under linux. The `TARGET` env var was for the ci script. I think you have to explicitly pass an arg to toolchain.

Reply:
Ah, yeah, I think you're right --- fixed in 610e411a579d17425f3f8ec810a28d38343ad2e4.